cmd/kops/integration_test.go

```diff
@@ -130,6 +130,11 @@ func TestSharedVPC(t *testing.T) {
 	runTestAWS(t, "sharedvpc.example.com", "../../tests/integration/shared_vpc", "v1alpha2", false, 1)
 }
 
+// TestSharedSecurityGroups runs the test on a configuration with a shared VPC
+func TestSharedSecurityGroups(t *testing.T) {
+	runTestAWS(t, "sec-groups.example.com", "../../tests/integration/shared_security_groups", "v1alpha2", false, 1)
```

Comment:
Might be easier to just add it into the 'complex' test ... not sure

Reply:
less sprawl is good

Reply:
Well, actually that would remove all the security group stuff in the complex example. So I would rather have this test separate and not included. I can do it either way. On the fence a bit. Have you a preference?